invokeai/backend/stable_diffusion/extensions_manager.py

```diff
@@ -63,9 +63,13 @@ def patch_extensions(self, context: DenoiseContext):
             yield None
 
     @contextmanager
-    def patch_unet(self, state_dict: Dict[str, torch.Tensor], unet: UNet2DConditionModel):
+    def patch_unet(self, unet: UNet2DConditionModel, cached_weights: Optional[Dict[str, torch.Tensor]] = None):
```

Comment:
`ExtensionBase.patch_unet(...)` needs to be updated to reflect this change in parameter ordering.